dash_docs/chapters/dash_vtk/index.py

```diff
@@ -0,0 +1,865 @@
+
+    - __vtkClass__:
+      - [vtkConcentricCylinderSource](https://kitware.github.io/vtk-js/api/Filters_Sources_ConcentricCylinderSource.html)
+      - [vtkConeSource](https://kitware.github.io/vtk-js/api/Filters_Sources_ConeSource.html)
+      - [vtkCubeSource](https://kitware.github.io/vtk-js/api/Filters_Sources_CubeSource.html)
+      - [vtkCylinderSource](https://kitware.github.io/vtk-js/api/Filters_Sources_CylinderSource.html)
+      - [vtkLineSource](https://kitware.github.io/vtk-js/api/Filters_Sources_LineSource.html)
+      - [vtkPlaneSource](https://kitware.github.io/vtk-js/api/Filters_Sources_PlaneSource.html)
+      - [vtkPointSource](https://kitware.github.io/vtk-js/api/Filters_Sources_PointSource.html)
+      - [vtkSphereSource](https://kitware.github.io/vtk-js/api/Filters_Sources_SphereSource.html)
+      - [vtkWarpScalar](https://kitware.github.io/vtk-js/api/Filters_General_WarpScalar.html)
+    - __state__: You will have to look on which property is available for your selected vtkClass.
+
+    The following example use a vtk source in vtk.js to generate a mesh
+    
+    '''),
+
+    rc.Markdown(
+        examples['t04_algorithm.py'][0], 
+        style=styles.code_container
+    ),
+
+    html.Div(
+        examples['t04_algorithm.py'][1], 
+        className='example-container'
+    ),
+
+
+    rc.Markdown('''
+
+    ### Reader
+
+    This element is similar to the __Algorithm__ one except that it focus on vtk.js readers by allowing to leverage their custom API.
+    A reader expect like an __Algorithm__ a __vtkClass__ among the one [listed below](https://github.com/Kitware/react-vtk-js/blob/master/src/AvailableClasses.js#L17-L24):
+
+    - __vtkClass__
+      - vtkPLYReader
+      - vtkSTLReader
+      - vtkElevationReader
+      - vtkOBJReader
+      - vtkPDBReader
+      - vtkXMLImageDataReader
+      - vtkXMLPolyDataReader
+
+    Then you should use one of the property below to feed the reader with some data:
+    - __url__: set of url to fetch data from (on the JS side)
+    - __parseAsText__: set the text content to process
+    - __parseAsArrayBuffer__: set binary data to process from base64 string
+
+    Since the data loading is going to be asynchronous we've enabled some automatic callback to either trigger a _render_ or a _resetCamera_ once the data became available. To enable those callback, just set the following set of properties to your licking:
+    - __renderOnUpdate__: True (default)
+    - __resetCameraOnUpdate__: True (default)
+
+    '''),
+
+    rc.Markdown(
+        examples['t05_reader.py'][0], 
+        style=styles.code_container
+    ),
+
+    html.Div(
+        examples['t05_reader.py'][1], 
+        className='example-container'
+    ),
+
+
+    rc.Markdown('''
+
+    ### ShareDataSet
+
+    This element does not affect the dataset or else, but it allow the JavaScript side to reuse an existing __vtkDataSet__ for another __Representation__ or __filter__.
```

Comment:
or else what? also allow -> allows

Reply:
```suggestion
    This element does not affect the dataset, but it allows the JavaScript side to reuse an existing __vtkDataSet__ for another __Representation__ or __filter__.
```

@jourdain Is the suggestion okay?

Reply:
yes